config/settings.yml

```diff
@@ -7,8 +7,7 @@ github_credentials:
   password:
 commit_monitor:
   bugzilla_product:
-gemfile_checker:
-  pr_contacts: []
+gem_changes_labeler:
```

Comment:
@bdunne Note this change...we will have to update the deploted settings.local.yml.